platform/viewer/src/connectedComponents/ToolbarRow.js

```diff
@@ -239,12 +258,34 @@ function _getVisibleToolbarButtons() {
   return toolbarButtonDefinitions;
 }
 
+/**
+ * Toggles the Download Dialog Modal
+ */
+function forceRenderUpdate() {
+  this.setState({
+    forceRenderUpdate: !this.state.isDownloadScreenShotDialogOpen,
+  });
+}
```

Comment:
I'm a little confused. Why would `forceRenderUpdate` toggle the DownloadDialogModal?

Reply:
@deprecated.